src/qt/overviewpage.cpp

```diff
@@ -153,15 +153,15 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 void OverviewPage::UpdateTip()
 {
     QStringList tips = {
-        tr("Never share your wallet.dat file/your private key with anyone"),
```

Comment:
I don't feel strongly about this, but I agree that users should not share their wallet files or private keys with anyone they don't want to have access to their transactions.

Are we losing any important nuance by dropping the mention of "your private key"?

(I think it's a mistake to get into the nuance of *why* you might provide emergency access to your wallet here, for what it's worth.The phrasing in a tip here is fine with me as it is.)

Reply:
There is another tip saying exactly the same thing on line 162 about the private key. I think that either we should keep one that mentions both, or keep two separate tips that anyway get rotated through.

I thought this to be the most elegant solution but can adapt to the former and remove line 162.

Reply:
No, you're right. It's better as you had it.